src/app/productConfig/productConfigForm/productConfigForm.component.spec.js

```diff
@@ -195,6 +195,15 @@ describe('product config form component', function () {
       expect($ctrl.analyticsFactory.giveGiftModal($ctrl.productData))
     })
 
+    it('should filter out givingLinks with empty urls', () => {
+      const validLink = { url: 'https://give.cru.org/1234567' }
+      jest.spyOn($ctrl.designationsService, 'givingLinks').mockReturnValue(Observable.of([{}, {}, validLink]))
```

Comment:
I think it would be nicer to have more realistic data in this test. An example empty item looks like this:

```
{
  name: undefined
  order: 0
  url: undefined
}
```

An example valid item would have all 3 properties.

Reply:
I ended up removing this test, but I made sure that new test uses data that looks like Jeff's in the test case.